pkg/envinfo/envinfo.go

```diff
@@ -421,6 +421,14 @@ func (ei *EnvInfo) GetApplication() string {
 	return ei.componentSettings.AppName
 }
 
+// MatchComponent matches a component information provided by a devfile compoment with the local env info
+func (ei *EnvInfo) MatchComponent(name, app, namespace string) bool {
+	if name == ei.GetName() && app == ei.GetApplication() && namespace == ei.GetNamespace() {
```

Comment:
It would be looking better as 
``` go
return name == ei.GetName() && app == ei.GetApplication() && namespace == ei.GetNamespace();
```
would it not?

Reply:
ack